scripts/exregional_run_analysis.sh

```diff
@@ -867,6 +902,20 @@ fi
 #-----------------------------------------------------------------------
 #
 # comment out for testing
+if [[ ${gsi_type} == "ANALYSIS" && ${ob_type} == "AERO" ]]; then
+
+if [ ${BKTYPE} -eq 1 ]; then
+ print_info_msg "$VERBOSE" "
+ Coldstart skip GSI SD DA"
+else
+
+  $APRUN ./gsi.x < gsiparm.anl > stdout 2>&1 || print_err_msg_exit "\
+  Call to executable to run GSI returned with nonzero exit code."
+
+fi
+
+else
+
```

Comment:
suggest removing line 905-918 and insert the following after line 643:

if [[ ${BKTYPE} -eq 1 ]]; then
   echo "cold start, skip GSI SD DA"
   exit 0
fi
(similar logic as line 826-831)